src/components/Book.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book, removeBook }) => {
+  const {
+    id,
+    title,
+    category,
+  } = book;
+
+  const handleRemoveBook = book => {
+    removeBook(book);
+  };
+
+  return (
+    <>
+      <td>{id}</td>
+      <td>{title}</td>
+      <td>{category}</td>
+      <td><button type="button" onClick={handleRemoveBook}>Remove Book</button></td>
```

Comment:
You could just use es6 to run the function directly.
```suggestion
      <td><button type="button" onClick={() => handleRemoveBook(book)}>Remove Book</button></td>
```